include/rtc/rtp.hpp

```diff
@@ -459,8 +459,9 @@ class RTCP_NACK {
 			return true;
 		} else {
 			// TODO SPEEED!
-			parts[(*fciCount) - 1].blp = htons(ntohs(parts[(*fciCount) - 1].blp) |
-			                                   (1u << (unsigned int)(missingPacket - *fciPID)));
+			auto blp = ntohs(parts[(*fciCount) - 1].blp);
```

Comment:
Is there a reason why `fciCount` and `fciPID` are passed by pointers and not by references in this function, since they may not be null? Also, why is `missingPacket` a const reference.

Reply:
The first two are personal preference as I prefer passing by pointer when that value is going to be modified. The last one just happened.